GOVERNANCE.md

```diff
@@ -55,6 +55,32 @@ The **PGB** also follows and is responsible for upholding the [OASIS Open Projec
 
 The PGB is comprised of one representative from each Sponsoring organization. Additionally, each TSC Chair is a member of the PGB. The PGB roster is maintained [here](https://github.com/eea-oasis/managed-open-project/blob/master/PROJECT-GOVERNING-BOARD.md)
 
+#### PGB Chairs
+
+The PGB must have a Chair or two co-Chairs. Only members of the committee are eligible to be Chair or co-Chair.  For the remainder of this section, Chair shall mean either the single Chair or each of the two co-Chairs, whichever the group has.
+
+The Chair is elected by Full Majority Vote of the committee. If the committee does not have a Chair, then all activities, with the exception of the selection of a new Chair, are suspended.
+
+Existing Chairs serve until they resign or are replaced in an election.  Elections should be held every two years.
+
+All PGB members, including the incumbent Chair, are eligible for nomination for the office.
+
+In the event of a vacancy (such as a co-Chair stepping down prior to the conclusion of their term), an election should be held for a replacement to complete the term.
+
+The process for electing the Chair shall be as follows:
+
+- The date for the Chair election shall be announced at least 14 days in advance on the PGB mailing list and copied to the general mailing list. The election shall take place during a regularly scheduled PGB meeting. An invitation to PGB members to nominate themselves or others for Chair must be included. PGB members may nominate themselves by sending an email to the PGB mailing list or in person during a PGB meeting.
+
+Where a member nominates a third party that nomination must be explicitly accepted by that third party, either verbally at a meeting or by email to the PGB
```

Comment:
minor nit on formatting: this either needs to be indented or marked as a separate bullet.

Reply:
Reformatted